packages/react-components/src/components/styles/SendBox.styles.ts

```diff
@@ -77,6 +77,10 @@ export const editorTextBoxButtonStyle: IStyle = {
  * @private
  */
 export const attachmentUploadCardsStyles = mergeStyles({
+  '& > *': {
+    // overwriting the background color set by Fluent UI
+    backgroundColor: 'rgb(0,0,0,0) !important'
```

Comment:
Could we try to do this without setting `important` modifier? 
Important modifier has the highest specificity and will override all style. Usually it's not a good thing for UI libraries

Please see https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity for more details about specificity